modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -224,138 +223,131 @@ xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in cas
+24. Put a new license in the database: see xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Runtime configuration] for further details.
+
+Below you can find a Linux exemple: +
+
 [source,bash]
 ----
  cd setup
  vi database.properties
  ./setup.sh pull
  ls -l ./platform_conf/licenses/
 ----
-+
+
 If there is no valid license in the `./platform_conf/licenses/`, these 2 pages will help you to request and install a new one:
 
  ** https://documentation.bonitasoft.com/?page=licenses[Licenses]
- ** xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Platform configuration]
+ ** xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Setup tool]
 
-+
 Install the new license:
-+
+
 [source,bash]
 ----
  cp BonitaSubscription-7.n-Jerome-myHosname-20171023-20180122.lic ./platform_conf/licenses/
  ./setup.sh push
 ----
 
-. Start the application server. Before you start Bonita Portal, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal pages instead of the new version.
- Log in to the Portal and verify that the migration has completed.
- If you did not set the default Look & Feel before migration and you cannot log in, you need to  xref:managing-look-feel.adoc[restore the default Look & Feel] using a REST client or the Engine API.
-. *If you migrated pasted version 7.7*
- In that case, if you used the migration tool 2.41.1 or greater, the table `arch_contract_data` is automatically backed up to the table `arch_contract_data_backup` to avoid long lasting migration.
- To reintegrate the data into your installation, a new tool is provided in versions 2.46.0 and above. It is located in the `tools/live-migration` folder.
- Follow instruction in the README.md to run this tool and re-integrate data from `arch_contract_data_backup`.
+25. Start the application server. Before you start Bonita Portal (before Bonita 2021.2) and Bonita Applications (from Bonita 2021.2), clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal or Applications pages instead of the new versions.
+26. Log in to Bonita UIs and verify that the updating procedure has completed.
+27. *If the target version is Bonita 7.7 or greater* and if you used Bonita Migration Tool 2.41.1 or greater, the table `arch_contract_data` is automatically backed up to the table `arch_contract_data_backup` to avoid long lasting migration.
+ To reintegrate the data into your installation, a new tool is provided in Bonita Migration Tool 2.46.0 and above. It is located in the `tools/live-migration` folder. Follow instruction in the README.md to run this tool and re-integrate data from `arch_contract_data_backup`.
+28. Custom Portal Look & Feel:
+ * If the target version is older than Bonita 2021.2: If you did not set the default Portal Look & Feel before updating and you cannot log in, you need to restore the default Look & Feel using a REST client or the Engine API.
```

Comment:
Same comment, do not talk about previous version if it doesn't apply to this version anymore.

Reply:
As a customer, it is ok to update to a version that is older than 2021.2, and this will be ok for two years still.

Reply:
removed

Reply:
removed